app/views/guidance/bulk_upload_placement_data.html.erb

```diff
@@ -0,0 +1,72 @@
+<%= render PageTitle::View.new(text: "Bulk upload placement data for trainee records") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: "How to use this service",
+    href: guidance_path,
+    ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l">Bulk upload placement data for trainee records</h1>
+
+    <p class="govuk-body-m">You can use Register trainee teachers (Register) to bulk upload placement data for trainee records.</p>
+
+    <h2 class="govuk-heading-m">Add placement data</h2>
+
+    <p class="govuk-body-m">You can bulk add placement data to registered trainee records.</p>
+
+    <p class="govuk-body-m">You can change placement details  <%= govuk_link_to("manually in an individual trainee’s record", manually_registering_trainees_guidance_path) %>.
+      You can also use the Higher Education Statistics Agency (HESA) service to update trainees in-year, if you have access to it.
+    </p>
+
+    <h2 class="govuk-heading-m">How to bulk upload placement data</h2>
+
+    <ol class="govuk-list govuk-list--number">
+      <li>Make sure you are signed into Register. Click into ‘Bulk updates’ in the main navigation.</li>
+      <li>Download a CSV file listing the trainees with missing placement data.</li>
+      <li>Add missing placement URNs.</li>
+      <li>Upload the CSV file</li>
```

Comment:
being pedantic, should we be consistent as to whether these bullet points include a full stop at the end?

Reply:
updated to be pedantic-ness compliant